### PR TITLE
Preserve music and genre data

### PR DESCRIPTION
**Pull request by V-Blaze**
In this branch, These are the significant changes:

- Implemented the `add_music_album`, `list_all_genres`, and `preserve_music_date` methods functionality.
- Take user inputs from the `cmd` to create `MusicAlbum` and `Genre` instances.
- Preserved and retrieve user data to and from a json file.

### CODE REVIEW

File: store/genre.json [hunk 1]
```diff
@@ -0,0 +1 @@
+["Hip hop","Comedy"]
```
Comment (yonas44) [inline store/genre.json:1]: Kindly, remove the redundant name of genre for a better readability😃👍

**Review by yonas44 — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @V-Blaze 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Genre class created :heavy_check_mark:
- MusicAlbum class created :heavy_check_mark:
- Correct Gitflow :heavy_check_mark:



# Optional Suggestions  :
- You can see some optional suggestions below

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: store/genre.json (hunk 1 above)
Comment (yonas44) [inline store/genre.json:1]: Kindly, remove the redundant name/ duplication of genre for a better readability😃👍

Comment (V-Blaze) [reply store/genre.json:1]: Thanks for the suggestion.. I'll make that changes in the next branch.. 😸

**Pull request merged**